src/webdriver_kaifuku/__init__.py

```diff
@@ -143,15 +138,17 @@ def _config_options_for_remote_firefox(browser_conf: dict) -> webdriver.FirefoxO
     @classmethod
     def from_conf(cls, browser_conf: dict) -> BrowserManager:
         browser_conf = copy(browser_conf)
+        browser_conf = _remove_deprecated_items(browser_conf)
+
+        browser_name = _get_browser_name(browser_conf)
         log.debug(browser_conf)
         webdriver_name = browser_conf.get("webdriver", "Firefox").title()
         webdriver_class = getattr(webdriver, webdriver_name)
 
         if webdriver_class not in TRUSTED_WEB_DRIVERS:
             log.warning(f"Untrusted webdriver {webdriver_name}, may cause failure.")
 
-        webdriver_kwargs = browser_conf.get("webdriver_options", {})
-        browser_name = _get_browser_name(webdriver_kwargs, webdriver_name)
+        webdriver_kwargs = dict()  # type: Dict[str, Any]
 
```

Comment:
Please use direct type annotations instead of comments

Reply:
@RonnyPfannschmidt sure, fixed now